src/index.html

```diff
@@ -1,13 +1,240 @@
+    <script src="scripts/main.js"
+            defer></script>
   </head>
+
   <body>
-    <h1>Miami</h1>
+    <header class="header">
+      <nav class="navigation">
+        <a class="navigation__item"
+           href="#">
+          <img src="../src/images/MyBiKE.svg"
+               alt="mybike logo">
+        </a>
+
+        <div class="navigation__icons">
+          <a class="navigation__item"
+             href="#">
+            <img src="../src/images/Icon-Phone-call.svg"
+                 alt="white phone-call" />
+          </a>
+
+          <a class="navigation__item"
+             href="#">
+            <img src="../src/images/Icon-Burger-menu.svg"
+                 alt="burger menu" />
+          </a>
+        </div>
+      </nav>
+      <h1 class="title">Take the Streets</h1>
+    </header>
+
+    <div class="menu">
+      <nav class="navigation">
+        <a class="navigation__item"
+           href="#">
+          <img src="../src/images/MyBiKE2.svg"
+               alt="mybike logo">
+        </a>
+
+        <div class="navigation__icons">
+          <a class="navigation__item--hidden"
+             href="#">
+            <img src="../src/images/Icon-Phone-call.svg"
+                 alt="white phone-call" />
+          </a>
+
+          <a class="navigation__item"
+             href="#">
+            <img src="../src/images/Icon-Close.svg"
+                 alt="burger menu" />
+          </a>
+        </div>
+      </nav>
+
+      <nav class="links menu__links">
+        <ul class="link__list">
+          <li class="links__item">
+            <a href="#"
+               class="links__link">Home
+            </a>
+          </li>
+          <li class="links__item">
+            <a href="#"
+               class="links__link">About Us
+            </a>
+          </li>
+          <li class="links__item">
+            <a href="#"
+               class="links__link">Compare Bikes
+            </a>
+          </li>
+          <li class="links__item">
+            <a href="#"
+               class="links__link">Details
+            </a>
+          </li>
+          <li class="links__item">
+            <a href="#"
+               class="links__link">Contacts
+            </a>
+          </li>
+        </ul>
+      </nav>
+
+      <p class="menu__paragraf">+1 234 5555-55-55</p>
+      <p class="menu__paragraf">
+        <a class="menu__call-link"
+           href="#">Book a test ride
+        </a>
+      </p>
+
+    </div>
+
+    <main class="main">
+      <h2 class="title title--secondary">Move Free</h2>
+      <p class="main__paragraf"> Shift your ride, not gears. Find the fastest way to move in the city as the bike adapts
+        intuitively to power the speed you need.
+      </p>
+
+      <section class="section">
+        <h2 class="title title--section">Compare bikes</h2>
+        <article class="article">
+          <img src="./images/photos/bike1.jpg"
+               class="pictures"
+               alt="bike sporty 4" />
```

Comment:
those images should be centered and have space before text
class should be due to BEM `article__picture`

Reply:
Formating will do it after finishing all elements. BEM done